### ISSUE TITLE
API BrewModifyEvent should include original ItemMeta

### ISSUE THREAD

**Issue by Heliosares** [DieReicheErethons/Brewery#471]
Please add the original ItemMeta or original ItemStack to BrewModifyEvent. I'm trying to find the original ItemStack to check something before the brew is sealed, but it's impossible with the current events.

Thank you

**Comment by Sn0wStorm** [comment]
Yes should be possible to attach the old item onto the event, where there is an old item.

**Comment by Sn0wStorm** [comment]
Ill consider adding this with the next feature release
